src/main/java/gov/usds/case_issues/model/CaseSnoozeSummaryFacade.java

```diff
@@ -44,6 +55,17 @@ public ZonedDateTime getSnoozeEnd() {
 		return wrapped.getSnoozeEnd();
 	}
 
+	public String getCreatedBy() {
```

Comment:
`@JSONIgnore` this one, as a short term fix—we shouldn't have the same value in two different places.

Alternatively, take it out entirely, and admit that this object no longer precisely extends CaseSnoozeSummary, which is maybe not great but we can probably live with in the short term.

Reply:
`getCreatedBy()` needs to be defined in `CaseSnoozeSummary` so we can grab the external user id in `findCaseDetails` https://github.com/usds/case-issue-api/pull/63/files#diff-6ee5cc833c1e5fc52b213b22759a6b31R72

added `@JSONIgnore` for now

Reply:
Yeah, gotcha—that just means that the API object for case snooze summaries and the DTO for those summaries have diverged, and we'll need to clean up.